VERSIONING.md

```diff
@@ -0,0 +1,145 @@
+# Versioning
+
+Universal Crossplane (UXP) uses a semantic versioning system based on the
+upstream Crossplane versioning to identify releases. UXP embeds Crossplane
+itself in its Helm chart, so the major, minor, and patch version of any UXP
+release will always match the embedded Crossplane major, minor, and patch
+version. The following sections describe the UXP versioning strategy, including
+when branches are cut, dependency tags are incremented, and new versions are
+released.
+
+## Format
+
+The UXP version string is formatted as a semantic version with appended
+pre-release information:
+
+```
+v<major>.<minor>.<patch>-up.<components>
+```
+
+The `<major>`, `<minor>`, and `<patch>` segments each refer to the corresponding
+embedded Crossplane `<major>`, `<minor>`, and `<patch>` versions. The embedded
+Crossplane version may differ from upstream Crossplane (i.e. Upbound may release
+patch versions past the end of life date for a Crossplane branch), but if the
+specified version exists upstream, the contents will match exactly.
```

Comment:
Good question -- We don't want any temporary fork to impact upstream release versioning (nor do we really have the power to do that). I imagine in this case that we would keep the Crossplane version the same and bump the `-up.X` version. This would allow for the next upstream patch release to be consumed with the presumed bugfix and for us to bump to the appropriate Crossplane version.